src/app/modules/dashboard/vault-stats/vault-stats.component.ts

```diff
@@ -0,0 +1,330 @@
+import {ChangeDetectorRef, Component, OnInit} from '@angular/core';
+import { ChartData } from './mokData/charts.data';
+import * as echarts from 'echarts';
+
+@Component({
+  selector: 'app-vault-stats',
+  templateUrl: './vault-stats.component.html',
+  styleUrls: ['./vault-stats.component.css']
+})
+export class VaultStatsComponent implements OnInit {
+
+    options1: any;
+    options2: any;
+
+    private data1: any[] = [];
+    private data2: any[] = [];
```

Comment:
no `any` please
create a model with a data structure that this component require